lib/get-config.js

```diff
@@ -303,7 +307,56 @@ function validateRules(config, options) {
   }
 }
 
-module.exports = function(options) {
+function _determineConfigForSeverity(config) {
+  switch (config) {
+    case 'off':
+      return { config: false, severity: IGNORE_SEVERITY };
+    case 'warn':
+      return { config: true, severity: WARNING_SEVERITY };
+    case 'error':
+      return { config: true, severity: ERROR_SEVERITY };
+  }
```

Comment:
Should we deal with `default` here also?

Reply:
the reason why I didn't want a `default` value here is that if one of the 3 cases is not found, it returns undefined which helps determine to use user-provided config instead.

Reply:
makes sense, thank you!